addons/l10n_dk/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_tax_report" model="account.tax.report.line">
```

Comment:
No need for a header.  However, in v14, we have the account.tax.report

Reply:
If necessary, we can remove it